docs/install/ipfs-updater.md

```diff
@@ -68,22 +77,31 @@ You can download pre-built binaries from [`dist.ipfs.tech`](https://dist.ipfs.te
    > -a----        11/5/2020   6:38 PM              0 Microsoft.PowerShell_profile.ps1
    ```
 
-7. Add the address you just copied to PowerShell's `PATH` by adding it to the end of the `Microsoft.PowerShell_profile.ps1` file stored in `Documents\WindowsPowerShell`:
+1. Add the address copied in step 5 to PowerShell's `PATH` by adding it to the end of the `Microsoft.PowerShell_profile.ps1` file stored in `Documents\WindowsPowerShell`:
 
    ```powershell
-   Add-Content C:\Users\Johnny\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1 "[System.Environment]::SetEnvironmentVariable('PATH',`$Env:PATH+';;C:\Users\Johnny\Apps\ipfs-update_v1.9.0\ipfs-update')"
+   Add-Content C:\Users\Johnny\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1 "[System.Environment]::SetEnvironmentVariable('PATH',`$Env:PATH+';;C:\Users\<username>\Apps\ipfs-update_v1.9.0\ipfs-update')"
```

Comment:
Should take the `\Johnny\` out of the first path line, change it to the variable `<username>` that is used everywhere else.